use_case_examples/llm/QGPT2Evaluate.ipynb

```diff
@@ -695,5 +695,5 @@
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
```

Comment:
Is this expected?

Reply:
No idea actually, `nbqa` probably? 🤷🏼

Reply:
Not sure. It's not important I guess.

Reply:
I didn't modify the file myself

Reply:
Can't do much about this one

Reply:
probably a version difference or something